test-fixtures/disconnecting-env/src/disconnecting-env.server.env.ts

```diff
@@ -1,14 +1,21 @@
 import testFeature, { serverEnv } from './disconnecting-env.feature';
 
-testFeature.setup(serverEnv, ({ run, errorType: { errorMode } }) => {
+testFeature.setup(serverEnv, ({ run, errorsConfig: { throwError, handleUncaught } }) => {
+    if (handleUncaught) {
+        process.on('uncaughtException', () => {
+            // User can do any other cleanup here
```

Comment:
What does this comment mean? How can the user do cleanup in a test fixture?

Reply:
this was an old comment, I've just refactored this fixture. but I agree, this is useless, so removed it